gst/nnstreamer/tensor_common.h

```diff
@@ -126,6 +126,41 @@ typedef enum _nns_media_type
   _NNS_MEDIA_END, /**< End Marker */
 } media_type;
 
+/**
+ * @brief Internal data structure for tensor element value.
+ */
+typedef struct
+{
+  tensor_type type;
+
+  /* tensor element data union for each tensor type */
+  union
+  {
+    TENSOR_TYPE (_NNS_INT32) v_int32;
+    TENSOR_TYPE (_NNS_UINT32) v_uint32;
+    TENSOR_TYPE (_NNS_INT16) v_int16;
+    TENSOR_TYPE (_NNS_UINT16) v_uint16;
+    TENSOR_TYPE (_NNS_INT8) v_int8;
+    TENSOR_TYPE (_NNS_UINT8) v_uint8;
+    TENSOR_TYPE (_NNS_FLOAT64) v_float64;
+    TENSOR_TYPE (_NNS_FLOAT32) v_float32;
+    TENSOR_TYPE (_NNS_INT64) v_int64;
+    TENSOR_TYPE (_NNS_UINT64) v_uint64;
+  } data;
+} GstTensorValue;
+
+/**
+ * @brief Define operators for tensor value.
+ */
+typedef enum
+{
+  TENSOR_VAL_OP_ADD = 0,
+  TENSOR_VAL_OP_MUL,
+  TENSOR_VAL_OP_DIV,
+
+  TENSOR_VAL_OP_UNKNOWN
+} tensor_value_op;
+
```

Comment:
Is this shared with other elements?
(otherwise, we don't need to export this.)

Reply:
Ok, I will move this definition and functions about tensor-value into tensor-transform.